test/mechanics/cell/PatchTest.cpp

```diff
@@ -132,18 +154,23 @@ BOOST_AUTO_TEST_CASE(PatchTestForce)
 
     IntegrationTypeTensorProduct<1> integrationTypeBc(1, eIntegrationMethod::GAUSS);
     Eigen::Vector2d pressureBC(1, 0);
-    Integrands::TimeDependent::NeumannBc<2> neumannBc(displ, pressureBC);
+    Integrands::NeumannBc<2> neumannBc(displ, pressureBC);
+    auto NeumannGradientF = Bind(neumannBc, &Integrands::NeumannBc<2>::Gradient);
+    auto NeumannHessian0F = Bind(neumannBc, &Integrands::NeumannBc<2>::Hessian0);
 
-    cellContainer.push_back(new Cell(boundaryElement, integrationTypeBc, neumannBc));
-    cellGroup.Add(cellContainer.back());
+    cellContainer.push_back(new Cell(boundaryElement, integrationTypeBc));
+    auto& neumannCell = cellContainer.back();
 
     // ************************************************************************
     //                  assemble and solve
     // ************************************************************************
     SimpleAssembler assembler(dofInfo.numIndependentDofs, dofInfo.numDependentDofs);
 
-    GlobalDofVector gradient = assembler.BuildVector(cellGroup, {&displ}, Integrands::TimeDependent::Gradient());
-    GlobalDofMatrixSparse hessian = assembler.BuildMatrix(cellGroup, {&displ}, Integrands::TimeDependent::Hessian0());
+    GlobalDofVector gradient = assembler.BuildVector(momentumBalanceCells, {displ}, MomentumGradientF);
+    gradient += assembler.BuildVector({neumannCell}, {displ}, NeumannGradientF);
+
+    GlobalDofMatrixSparse hessian = assembler.BuildMatrix(momentumBalanceCells, {displ}, MomentumHessian0F);
+    hessian += assembler.BuildMatrix({neumannCell}, {displ}, NeumannHessian0F);
```

Comment:
Remark from me. This is not an optimal solution. Adding two SparseMatrices with possibly different sparsity patterns can be very expensive. I added this issue to [cleanup after completion](https://github.com/nutofem/nuto/projects/1).